app/models/custom_filter.rb

```diff
@@ -17,8 +17,16 @@
 #  index_custom_filters_on_user_id     (user_id)
 #
 class CustomFilter < ApplicationRecord
+  MAX_FILTER_PER_USER = 50
```

Comment:
Should we make this a global config?

Reply:
@scmmishra We can, but if the admin changes it to >50, it will be hard to handle the filter considering the performance.

Let's revisit this when the filter is ready and change it according to the performance.
Right now, we need a clear idea about the same.

Reply:
Sounds good